### ISSUE TITLE
Still alive?

### ISSUE THREAD

**Issue by supermarcos** [gueei/AndroidBinding#9]
Hi, there!

Are you still supporting and developing this library?
Because I found a few places with getting-started tutorials and I can see they must be old or something because using the current jar is published here (BTW: is that library compiled with the last codebase?) things seem to be different...

Thanks

**Comment by leojh** [comment]
I would like to know this as well, please.

**Comment by xudshen** [comment]
+1

**Comment by leojh** [comment]
Everyone checkout https://github.com/RoboBinding/RoboBinding for a data-binding lib that is alive and works great.

**Comment by supermarcos** [comment]
Hi, leojh, have you been able to put RoboBinding in your project? Because I already tried and I couldn't, the issues with gradle are a nightmare. Did you put it in a project using Android Studio? Thanks!

**Comment by leojh** [comment]
Yes, I was able to use it on a prototype that I built. I agree the gradle stuff is a nightmare overall specially since I first tried it out using plain old InteliJ IDEA. However, the latest version of RoboBinding has the gradle config built in for Android Studio.

**Comment by egandro** [comment]
I use Roboguice with Android Binding. Maybe I'll add a demoproject. Basically you need your own Roboguice + Binding Activity merged into one class.

**Comment by egandro** [comment]
And yes the library is still supprted. I'll add some fixes and some new stuff during the next days.
